apps/dataset-browser/src/app/error.tsx

```diff
@@ -0,0 +1,15 @@
+'use client'; // Error components must be Client components
+
+import {useEffect} from 'react';
+
+export default function Error({error}: {error: Error}) {
+  useEffect(() => {
```

Comment:
Curious: do we need a `useEffect` here? What if we console.log the error without it?

Reply:
I got this from the Next.js docs: https://beta.nextjs.org/docs/routing/error-handling

I think it's to prevent multiple console logs if there are any rerenders. Maybe from the layout. But in our case, I don't think it's necessary. So I will remove it.